lib/modules/platform/gitlab/index.ts

```diff
@@ -1340,3 +1346,22 @@ export async function filterUnavailableUsers(
   }
   return filteredUsers;
 }
+
+export async function expandGroupMembers(
+  reviewersOrAssignees: string[]
+): Promise<string[]> {
+  const expandedReviewersOrAssignees: string[] = [];
+  for (const reviewerOrAssignee of reviewersOrAssignees) {
+    try {
+      const members = await getMemberUsernames(reviewerOrAssignee);
+      expandedReviewersOrAssignees.push(...members);
+    } catch (err) {
+      // Unable to fetch group or is not a group, so add back to list.
```

Comment:
isn't there a better way to differentiate between user and group?

Reply:
See the below comment, but we can have user@email.com, @username, and @group. We can filter everything w/o an @ before, but will still need to see if the API request will fail for when an @username is provided.